experimental/iterators/include/operators/column_scan.h

```diff
@@ -0,0 +1,50 @@
+#ifndef OPERATORS_COLUMN_SCAN_H
+#define OPERATORS_COLUMN_SCAN_H
+
+#include <optional>
+#include <tuple>
+#include <vector>
+
+template <typename... InputTypes>
+class ColumnScanOperator {
+ public:
+  using OutputTuple = std::tuple<InputTypes...>;
+  using ReturnType = std::optional<OutputTuple>;
+
+  explicit ColumnScanOperator(std::vector<InputTypes>... inputs)
+      : inputs_(std::make_tuple(std::move(inputs)...)), current_pos_(0) {}
+
+  void Open() {}
+  ReturnType ComputeNext() {
```

Comment:
one (the main) implication of the LLVM style here would be that the functions and variables are all snakeCase.

Reply:
Dito: cheapest to fix now. The latest commit applies all things I realized deviated from the LLVM style guide. Let me know if you spot anything else (or I misunderstood the guide or whatever).